### DISCUSSION TITLE
Найцікавіші новини місяця: січень 2024

### DISCUSSION THREAD

**Discussion by rkiyanchuk** [shopokodu/community#48] [Теми для подкасту]
- [x] [Apple announces changes to iOS, Safari, and the App Store in the European Union](https://www.apple.com/newsroom/2024/01/apple-announces-changes-to-ios-safari-and-the-app-store-in-the-european-union/?utm_source=hackernewsletter&utm_medium=email&utm_term=fav)
   * https://commission.europa.eu/strategy-and-policy/priorities-2019-2024/europe-fit-digital-age/digital-markets-act-ensuring-fair-and-open-digital-markets_en
- [x] [Maximum size of a PDF file](https://alexwlchan.net/2024/big-pdf/?utm_source=hackernewsletter&utm_medium=email&utm_term=fav)
- [x] The [Mammoth2024h1 CT log died](https://groups.google.com/a/chromium.org/g/ct-policy/c/038B7F4g8cU) because the server ran out of disk space, a situation that led to catastrophic MySQL failure. Maybe Trillian (Google’s CT log implementation) should have been built around PostgreSQL instead?
   * https://certificate.transparency.dev/howctworks/
- [x] [What if I paid for all my free software?](https://www.cynicusrex.com/file/takemymoney.html)
- [ ] [Why does the community always seem to choose Red Hat over Canonical?](https://www.reddit.com/r/linux/comments/18yvefc/why_does_the_community_always_seem_to_choose_red/)
- [x] [Princeton researchers say generative AI isn't replacing devs any time soon](https://www.reddit.com/r/programming/comments/1aelwr1/princeton_researchers_say_generative_ai_isnt/)
- [ ] [Nearly 25,000 tech workers were laid off in the first weeks of 2024. Why is that?](https://www.reddit.com/r/programming/comments/1adfdq2/nearly_25000_tech_workers_were_laid_off_in_the/)
- [ ] [OpenWRT wants to offer its own router](https://www.reddit.com/r/linux/comments/1936i1j/openwrt_wants_to_offer_its_own_router/)
- [ ] [Framework Laptop 16 review: The perfect laptop for PC hardware nerds](https://www.pcworld.com/article/2215545/framework-laptop-16-review.html)
- [x] [JetBrains' unremovable AI assistant meets irresistible outcry](https://www.theregister.com/2024/02/01/jetbrains_unremovable_ai_assistant/)

**Comment by yakimka** [comment]
[The Day Before was an even bigger disaster than you thought: devs reportedly made to pay fines for bad work, learned it was an MMO from the trailers, and no one's sure where the bosses are](https://www.pcgamer.com/the-day-before-was-an-even-bigger-disaster-than-you-thought-devs-reportedly-made-to-pay-fines-for-bad-work-learned-it-was-an-mmo-from-the-trailers-and-no-ones-sure-where-the-bosses-are/) - Для контексту: The Day Before це гра від руснявої студії
